app/models/participant_profile.rb

```diff
@@ -0,0 +1,41 @@
+# frozen_string_literal: true
+
+class ParticipantProfile < ApplicationRecord
```

Comment:
Would you consider calling this something like `ParticipationRecord`? We already have the idea of a profile, and use it to mean something else

Reply:
Ok. Maybe we don't fully grok your use of Profile.

Reply:
Done